neuralmonkey/encoders/sentence_encoder.py

```diff
@@ -1,139 +1,211 @@
+# tests: lint, mypy
+
+from typing import Optional, Dict, Any, Tuple
+
 import tensorflow as tf
-import numpy as np
 
 from neuralmonkey.logging import log
 from neuralmonkey.nn.noisy_gru_cell import NoisyGRUCell
-from neuralmonkey.nn.pervasive_dropout_wrapper import PervasiveDropoutWrapper
 from neuralmonkey.nn.ortho_gru_cell import OrthoGRUCell
-from neuralmonkey.checking import assert_type
+from neuralmonkey.dataset import Dataset
 from neuralmonkey.vocabulary import Vocabulary
 
+# pylint: disable=invalid-name
+AttType = Any ## Type[] or union of types do not work here
+FeedDict = Dict[tf.Tensor, Any]
```

Comment:
Já přemejšlel vůbec udělat modul `neuralmonkey.typing`, kde to bude všechno a odkud se to bude všechno importovat